src/main/java/de/unistuttgart/towercrushbackend/controller/components/WebsocketListener.java

```diff
@@ -122,7 +123,7 @@ private void handleDeveloperJoined() {
                     try {
                         Thread.sleep(1000);
                     } catch (final Exception e) {
-                        throw new RuntimeException(e);
+                        throw new InterruptedException();
                     }
```

Comment:
Erm…
That won't work for two reasons:
1. `Thread.sleep` already throws `InterruptedException`
2. That is a checked exception that you cannot throw here.